provider/inspec.rb

```diff
@@ -157,5 +159,12 @@ def nested_object_requires(nested_object_type)
         [nested_object_type.__resource.name, nested_object_type.name.underscore].join('_')
       ).downcase
     end
+
+    def plural(word)
+      # TODO: use a real ruby gem for this? Pluralization is hard
```

Comment:
Hm.  I'm inclined to ask you to do this, yeah - is it going to cause problems to change these things later?  Are these user-visible or in user configs?

Reply:
They end up being user visible. Used in both the names of resources (these seem to work fine with this method) and names of attributes on resources (this doesn't work so well. For example pluralizing `disks` -> `diskses`)

Reply:
haha, `tricksy_hobbitses`

Please go ahead and use a real ruby gem for plurals if you can - if it'd be a _ton_ of work, feel free to skip it, but it'd be better to have it right the first time.